kurtosis-devnet/tests/eigenda/failover_test.go

```diff
@@ -313,7 +341,7 @@ func getEndpointsFromKurtosis(enclaveCtx *enclaves.EnclaveContext) (*EnclaveServ
 		// Get the kurtosis tag
 		tag := field.Tag.Get("kurtosis")
 		if tag == "" {
-			continue // Skip fields without tags
+			return nil, fmt.Errorf("field %s doesn't have a kurtosis tag", field.Name)
```

Comment:
If we require all tags to have `kurtosis` tag, can we just drop the tag requirement altogether?